### PR TITLE
DOC: use latex for math

### PR DESCRIPTION
**Pull request by sciunto**
Looks better, isn't it?

There was a mix of wrong math tags and missing tags.

### CODE REVIEW

File: skimage/transform/_geometric.py [hunk 1]
```diff
@@ -226,15 +226,21 @@ def estimate(self, src, dst):
...
 
-        The transformation is defined as::
+        The transformation is defined as:
 
-            X = (a0*x + a1*y + a2) / (c0*x + c1*y + 1)
-            Y = (b0*x + b1*y + b2) / (c0*x + c1*y + 1)
+        .. math::
 
-        These equations can be transformed to the following form::
+            X = (a_0\,x + a_1\,y + a_2)/(c_0\,x + c_1\,y + 1)
```
Comment (soupault) [inline skimage/transform/_geometric.py]: Nice! At first, I was thinking about \cdot, but this looks much better. Maybe even use \frac here?

**Comment by soupault** [comment]
@sciunto I have one comment, but the PR looks good to me as is. :+1:

File: skimage/transform/_geometric.py (hunk 1 above)
Comment (sciunto) [inline skimage/transform/_geometric.py]: Yeah, I tried, but didn't work. I guess there is a sphinx super-trick.

Comment (soupault) [inline skimage/transform/_geometric.py]: It should work, [apparently](http://matplotlib.org/sampledoc/extensions.html).
Have you tried this https://stackoverflow.com/questions/16468397/mathjax-expression-in-sphinx-python-not-rendering-correclty ?

**Comment by sciunto** [comment]
OK, we are good now :) Thanks.

**Comment by ahojnnes** [comment]
Hm... I am not sure if we want this. I for myself look at the doc strings in text mode and do not use the HTML documentation. This is close to unreadable in text format...

**Comment by sciunto** [comment]
OK i close then.

**Pull request closed without merging**